.env.template

```diff
@@ -1,3 +1,13 @@
 GIT_SSH_COMMAND='ssh -i /var/.ssh_keys/github_deploy_key -o IdentitiesOnly=yes -o StrictHostKeyChecking=no'
-ENV=development
-DEVELOPER_USERNAME=sampleusername
+
+TMS = /var/tms
```

Comment:
optional - if we can name this `TMS_PATH`
and the one below `TMS_DATA_PATH` that would be a bit more explicit, but mostly a matter of preference not a need to change 👍

Reply:
Agreed! I've updated it with those changes.